themes/default/content/product/_index.md

```diff
@@ -12,80 +12,115 @@ subheading: |
-                    encapsulate your cloud architectures and best practices.
+                    Run rapid unit tests with any supported framework to validate your infrastructure code before deployment.
 
-        - description: Deploy cloud infrastructure and applications together
+        - description: Consume infrastructure as code in any language
           features:
-              - title: Multiple deployment options
+              - title:
                 icon: rocketship
                 icon_color: salmon
                 description: |
-                    Deploy infrastructure interactively with a CLI, programmatically with Pulumi’s
-                    [Automation API](/automation/), or through your [CI/CD process](/docs/guides/continuous-delivery/).
+                    Build faster by reusing Pulumi libraries and CDK constructs, and create your own libraries as multi-language Pulumi Packages.
 
-              - title: Deployments as Code
+              - title:
                 icon: gear
                 icon_color: salmon
                 description: |
-                    Run deployments from your application code at runtime with [Automation API](/automation/). Create infrastructure APIs, custom platforms, and CLIs.
+                    Develop modern cloud applications faster while using well-architected best practices with the Crosswalk for AWS library.
 
-              - title: Preview and test changes
+              - title:
                 icon: eye
                 icon_color: salmon
                 description: |
-                    Test and validate infrastructure with standard [unit test frameworks](/docs/guides/testing/#unit-testing) and
-                    [integration tests](/docs/guides/testing/integration/). Preview changes before deploying.
+                    Convert your existing infrastructure code to Pulumi (TF, ARM, CFN, K8s, CRD, Helm) and import existing resources into Pulumi code.
+     
+        - description: Deploy to over 100 cloud and SaaS providers
+          features:
+              - title: 
+                icon: shield
+                icon_color: purple
+                description: |
+                    Native providers for AWS, Azure, Google Cloud, and Kubernetes have 100% API coverage and same-day updates for new features.
+
+              - title: 
+                icon: security
+                icon_color: purple
+                description: |
+                    Use a single workflow to manage cloud native infrastructure like Kubernetes, Helm, CRDs, GitOps, and more.
 
-        - description: Manage cloud applications with visibility and controls
+              - title: 
+                icon: monitor
+                icon_color: purple
+                description: |
+                    Find 100+ providers and cloud components in the Pulumi Registry. Bridge any existing Terraform provider to Pulumi.
```

Comment:
link Pulumi Registry to the registry
link "bridge" to https://github.com/pulumi/pulumi-tf-provider-boilerplate